packages/cli/package.json

```diff
@@ -36,7 +36,7 @@
     "@ast-grep/napi": "0.11.0",
     "@oclif/core": "3.26.5",
     "@shopify/cli-kit": "3.60.0",
-    "@shopify/oxygen-cli": "~4.4.5",
+    "@shopify/oxygen-cli": "4.4.9",
```

Comment:
We have added `~` and removed it multiple times. Now that we are bundling the CLI I think it's better to be more intentional with this because oxygen-cli might inadvertedly bump some subdependency and become incompatible with bundling (e.g. oclif, cli-kit, etc.).

Plus, the `~` doesn't really change anything because we are locked by package-lock.json, so we only update it if we do it manually... or by mistake if something else updates.